README.md

```diff
@@ -24,7 +24,7 @@ Want to know more?
 ### Quick Start with Heroku
 This version of Spoke suitable for testing and, potentially, for small campaigns. This won't cost any money and will not support production(aka large-scale) usage. It's a great way to practice deploying Spoke or see it in action.  
 
-<a href="https://heroku.com/deploy?template=https://github.com/MoveOnOrg/Spoke/tree/v12.1">
+<a href="https://heroku.com/deploy?template=https://github.com/MoveOnOrg/Spoke">
```

Comment:
```suggestion
<a href="https://heroku.com/deploy?template=https://github.com/MoveOnOrg/Spoke/tree/12.1.0">
```

Reply:
Hey @crayolakat, I returned the /tree... and made sure to reformat the version number.  You should see those changes reflected now.